README.md

```diff
@@ -97,6 +97,16 @@ This matches https://github.com/rhboot/shim/releases/tag/15.5 and contains the a
 * CVE-2020-15705
 * CVE-2021-3418 (if you are shipping the shim_lock module)
 
+* CVE-2021-3695
+* CVE-2021-3696
+* CVE-2021-3697
+* CVE-2022-28733
+* CVE-2022-28734
+* CVE-2022-28735
+* CVE-2022-28736
+* CVE-2022-28737
+
```

Comment:
There's an extra newline both before and after the CVE block (i.e., here and line 99).